.github/workflows/release.yml

```diff
@@ -0,0 +1,17 @@
+on:
+  push:
+    tags: "*"
+
+jobs:
+  push_to_dockerhub:
+    strategy:
```

Comment:
I think it will be a little bit clearer and provide an additional description for the job like below:
jobs:
  push_to_dockerhub:
    name: "Push Docker Images to DockerHub"
    strategy: ...